docs/index.md

```
@@ -154,6 +154,23 @@ spec:
     </div>
 </div>
 
+<div class="get-started-section">
+  <h2>Demo - See KubeElasti in action!</h2>
+<div style="position: relative; padding-bottom: 55.026178010471206%; height: 0;">
+
+<iframe 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)